app/src/main/java/com/example/felipe/criminalintent/Crime.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.felipe.criminalintent
+
+import java.util.*
```

Comment:
be careful of this import right here. I know it's not your fault, but in the future this will make the linter fail your build. You should only import what you need. This happened because the optimize code feature for Kotlin doesn't work too well and adds these kind of imports (in a Java project it works OK), it's a pain in the ass, I know, just be aware that this is a thing

Reply:
Yo can add a rule in the ide configuration to avoid it :smile:

Reply:
Jesus christ, why are you telling me just now??? hahahaha